ts/Series/Map/MapSeries.ts

```diff
@@ -1551,6 +1551,8 @@ export default MapSeries;
  *
  * @sample maps/series/data-geometry/
  *         Geometry defined in data
```

Comment:
Maybe renaming the original demo to its title "Map point and line geometry" could make it easier to know what is the difference between those demos just from the name in the API docs?